mod part1_2;
use common::error::Result;

fn main() -> Result<()> {
    part1_2::run()?;
    Ok(())
}
